scrapy/extensions/corestats.py

```diff
@@ -21,10 +22,16 @@ def from_crawler(cls, crawler):
         return o
 
     def spider_opened(self, spider):
-        self.stats.set_value('start_time', datetime.datetime.utcnow(), spider=spider)
```

Comment:
I don't think moving from UTC time to a local time is a good idea, due to the following:
- this is a backwards incompatible change, we can't just change this
- time zone on the server might be different from the user time zone anyway, so using UTC seems less confusing to me - this is just my opinion though

Reply:
@lopuhin Yeah I agree with you and I have made the necessary changes! :)

Reply:
@maramsumanth code looks nice, thanks! 👍

Reply:
@lopuhin , Is it necessary to add tests?